app/features/build/views/build.erb

```diff
@@ -31,6 +31,23 @@
       </a>
       </td>
     </tr>
+    <% if build.git_fork_config %>
+      <% if build.git_fork_config.branch.to_s.length > 0 %>
```

Comment:
This is a nitpick, but could we potentially use [`positive?`](https://ruby-doc.org/core-2.3.0/Numeric.html#method-i-positive-3F) instead? If you prefer `> 0`, that's cool too.

So:

```erb
<% if build.git_fork_config.branch.to_s.length.positive? %>
```

Reply:
Thanks, yes, I prefer `> 0`, as it's easier to read/parse when skimming through source code

Reply:
Sounds good, I'll use that convention too from now on. :)